src/ExtInterface/ADRIA/Domain.jl

```diff
@@ -18,13 +18,13 @@ mutable struct ADRIADomain <: Domain
     env_layer_md::EnvLayer  # Layers used
     scenario_invoke_time::String  # time latest set of scenarios were run
     const conn::YAXArray  # connectivity data
-    site_data::DataFrame  # table of site data (depth, carrying capacity, etc)
-    const site_id_col::String  # column to use as site ids, also used by the connectivity dataset (indicates order of `conn`)
-    const cluster_id_col::String  # column of unique site ids
+    site_data::DataFrame  # table of location data (depth, carrying capacity, etc)
+    const site_id_col::String  # column to use as location ids, also used by the connectivity dataset (indicates order of `conn`)
+    const cluster_id_col::String  # column of unique cluster ids
     init_coral_cover::YAXArray  # initial coral cover dataset
     const coral_growth::CoralGrowth  # coral
-    const site_ids::Vector{String}  # Site IDs that are represented (i.e., subset of site_data[:, site_id_col], after missing sites are filtered)
-    const removed_sites::Vector{String}  # indices of sites that were removed. Used to align site_data, DHW, connectivity, etc.
+    const site_ids::Vector{String}  # Location IDs that are represented (i.e., subset of site_data[:, location_id_col], after missing locations are filtered)
```

Comment:
```suggestion
    const site_ids::Vector{String}  # Location IDs that are represented (i.e., subset of site_data[:, loc_id_col], after missing locations are filtered)
```

Standardize to `loc_id_col`?